Makefile

```
@@ -100,12 +106,17 @@ build:
 	mkdir -p bin/ && ${GOENV} ${GO} build ${GOBUILDFLAGS} -ldflags "-X cmd.Version=$(VERSION)" -o ./bin/ ./...
 
 .PHONY: docker-build-push
-docker-build-push:
-	./build_deploy.sh
-
-.PHONY: build-push-minimal
-build-push-minimal:
-	./build_push_minimal.sh
+docker-build-push: ## Build and push the container image; IMAGE and quay.io login are required
```

checkmake (0.3.2)

[warning] Target body for "docker-build-push" exceeds allowed length of 5 lines (11).

(maxbodylength)